utils/i18nUtils.ts

```diff
@@ -1,13 +1,14 @@
 /* eslint-disable simple-import-sort/imports */
-import { LinkPropertyHref } from 'vue-meta'
-import type { Locale, LocaleObject } from 'nuxt-i18n'
+import type { NuxtOptionsHead as MetaInfo } from '@nuxt/types/config/head'
```

Comment:
```suggestion
import { LinkPropertyHref } from 'vue-meta'
```
eslintの `import/named` のエラーがでますがここは元のままでよいようです